pennylane/devices/qubit/apply_operation.py

```diff
@@ -21,6 +21,7 @@
 import pennylane as qml
 
 from pennylane import math
+from ..default_qubit import MidMeasureMP
```

Comment:
Why not from `pennylane/meausrements/` ? Might be safer in terms in terms of circular dependencies.

Reply:
No reason in particular, let me change it.